docker-rancher/nodes.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "rancherApi": {
+      "type": "string",
+      "metadata": {
+        "description": "The api link to your RancherHost. Example : http://myrancherhost.kvaes.be:8080/v1/scripts/AA12A123A1A1234A12A1:1234567890000:aAaaaaA1AaAaAlaAAaaaaAAAaaA"
+      }
+    },
+	"rancherNodeName": {
+      "type": "string",
+      "metadata": {
+        "description": "The prefix you want to use as hostname"
+      }
+    },
+	"rancherVmSize": {
+	  "type": "string",
+	  "metadata": {
+	    "description": "The size of the virtual machine used for the nodes"
+	  },
+	  "allowedValues": [
+		"Standard_A0",
+		"Standard_A1",
+		"Standard_A2",
+		"Standard_A3",
+		"Standard_A4",
+		"Basic_A0",
+		"Basic_A1",
+		"Basic_A2",
+		"Basic_A3",
+		"Basic_A4",
+		"Standard_A5",
+		"Standard_A6",
+		"Standard_A7",
+		"Standard_D1",
+		"Standard_D2",
+		"Standard_D3",
+		"Standard_D4",
+		"Standard_D11",
+		"Standard_D12",
+		"Standard_D13",
+		"Standard_D14",
+		"Standard_D1_v2",
+		"Standard_D2_v2",
+		"Standard_D3_v2",
+		"Standard_D4_v2",
+		"Standard_D5_v2",
+		"Standard_D11_v2",
+		"Standard_D12_v2",
+		"Standard_D13_v2",
+		"Standard_D14_v2",
+		"Standard_DS1",
+		"Standard_DS2",
+		"Standard_DS3",
+		"Standard_DS4",
+		"Standard_DS11",
+		"Standard_DS12",
+		"Standard_DS13",
+		"Standard_DS14",
+		"Standard_G1",
+		"Standard_G2",
+		"Standard_G3",
+		"Standard_G4",
+		"Standard_G5",
+		"Standard_GS1",
+		"Standard_GS2",
+		"Standard_GS3",
+		"Standard_GS4",
+		"Standard_GS5",
+		"Standard_A8",
+		"Standard_A9",
+		"Standard_A10",
+		"Standard_A11"
+	  ],
+	  "defaultValue": "Standard_A2"
+	},
+	"rancherCount": {
+      "type": "int",
+      "metadata": {
+        "description": "The amount of nodes to be provisioned"
+      },
+	  "minValue": 1,
+	  "maxValue": 250
+    }
+  },
+  "variables": {
+    "apiVersion": "2015-05-01-preview",  
+    "availabilitySet": "[parameters('rancherNodeName')]",
+    "storageAccountName": "[parameters('rancherNodeName')]",
+    "networkSecurityGroupName": "[parameters('rancherNodeName')]",
+    "imagePublisher": "Canonical",
+    "imageOffer": "UbuntuServer",
+    "ubuntuOSVersion": "15.10",
```

Comment:
Can this be a LTS release? The 15.10 image will be removed from Azure when 15.10 goes EOL and this template will break